auth/README.md

```diff
@@ -0,0 +1,215 @@
+# YMCA Authentication Server
+
+    ymca-auth.now.sh
+
+Made from [H4i Infrastructure Authentication Server](https://github.com/hack4impact-uiuc/infra-authentication-server).
+
+# Permission Structure
+
+**admin** can promote or demote:
+ - *intern*
+ - *public*
+
+**intern** can promote or demote:
+ - *public*
+
+**public** has no ability to promote or demote.
+
+# Links
+
+Documentation: https://www.notion.so/h4iuiuc/Authentication-Server-12c9fa26d1b9457bb84d7105caea530e
+
+API: https://github.com/hack4impact-uiuc/infra-authentication-api/
+
+Guide for integrating into your App: https://docs.google.com/document/d/1K6e9jarVtAync-Bti6BN6bKI-8JnvwK4IZmhlTSn2pg/edit
+
+Client Example: https://github.com/hack4impact-uiuc/infra-authentication-client
+
+# To Run Locally
+
+```
+yarn
+yarn start
+```
+
+# Deployment
+
+## AWS Elastic Beanstalk
+
+1. Create an AWS account and go to the [Elastic Beanstalk website](https://console.aws.amazon.com/elasticbeanstalk/home) (note that you may be charged)
+2. Create a new Elastic Beanstalk application, along with a web server environment
+   ![](./dimages/eb1.png)
```

Comment:
You can upload all the images to Github and remove the local image files. So then instead of `./dimages/eb1.png` you can put in the Github URL. To upload, open an issue in a repo and just drag and drop the images into the text box. It'll automatically upload and give you the URL. Make sure not to submit the issue :)